src/appleseed.python/bindblenderprogressivetilecallback.cpp

```diff
@@ -214,9 +217,14 @@ namespace
                     if (shader_count < 2)
                         create_vertex_shader();
 
+                    glUniformMatrix4fv(glGetUniformLocation(m_shader_program_id, "ModelViewProjectionMatrix"), 1, GL_TRUE, &(Matrix4f::make_identity()[0]));
```

Comment:
`Matrix4f::make_identity()` _creates_ an identity matrix, but you can use `Matrix4f::identity()` which returns a "precomputed" identity matrix.

Reply:
actually, I forgot to delete this line. We don't need it.